echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/NotificationTemplateEngine.groovy

```diff
@@ -129,4 +136,20 @@ class NotificationTemplateEngine {
         return renderer.render(document)
       }
     }
+
+  static class MarkdownPassThruFormatter implements Formatter {
```

Comment:
I found out that, probably due to recent updates to how the Slack clients handle rendering of text, the default formatter above was causing the client to display HTML code (e.g. `<em>bold</em>`) instead of rendering the original markdown (`*bold*`) properly.